src/main/java/org/folio/inventory/resources/AbstractInstances.java

```diff
@@ -71,11 +72,11 @@ public AbstractInstances(final Storage storage, final HttpClient client) {
     CollectionResourceClient relatedInstancesClient = createInstanceRelationshipsClient(
       routingContext, context);
     CollectionResourceRepository relatedInstancesRepository = new CollectionResourceRepository(relatedInstancesClient);
-    List<String> instanceId = Arrays.asList(instance.getId());
+    List<String> instanceId = Collections.singletonList( instance.getId() );
     String query = createQueryForRelatedInstances(instanceId);
 
     CompletableFuture<Response> future = new CompletableFuture<>();
-    relatedInstancesClient.getMany(query, future::complete);
+    relatedInstancesClient.getMany(query, Integer.MAX_VALUE, 0, future::complete);
```

Comment:
The magic values `Integer.MAX_VALUE, 0` should be avoided by adding a method like getAll(String query, Consumer<Response>) to CollectionResourceClient. This also avoids duplication of this code in Instances.java.

Javadoc should be added to CollectionResourceClient.getMany pointing out the most APIs be default have a limit of 10 records.